### PR TITLE
removed unnecessary javascript to reduce web bloat

### PR DESCRIPTION
**Pull request by bermannoah**
our score [here](http://www.webbloatscore.com/) is 4.36  -- would be ideal for it to be under 1.00

basically just removed a ton of javascript! nothing has blown up yet.

### CODE REVIEW

File: app/views/coffee/index.html.erb [hunk 1]
```diff
@@ -43,6 +43,6 @@
   @noah on slack with questions! hope you enjoy :)
 </p>
 
-<a href="https://twitter.com/turingcoffeebot" class="link-decoration" data-show-count="false">follow @turingcoffeebot</a><script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+<a href="https://twitter.com/turingcoffeebot" class="link-decoration" data-show-count="false" target="_blank">follow @turingcoffeebot</a>
```
Comment (NickyBobby) [inline app/views/coffee/index.html.erb]: Is this link blank for right now?

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -1,15 +0,0 @@
...
-//
-// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
-// or any plugin's vendor/assets/javascripts directory can be referenced here using a relative path.
-//
-// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
-// compiled file. JavaScript code in this file should be added after the last require_* statement.
-//
-// Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
-// about supported directives.
-//
```
Comment (NickyBobby) [inline app/assets/javascripts/application.js]: You really ain't down with the JS, huh?

File: app/views/coffee/index.html.erb (hunk 1 above)
Comment (NickyBobby) [inline app/views/coffee/index.html.erb]: Ok kewl

File: app/assets/javascripts/application.js (hunk 2 above)
Comment (bermannoah) [inline app/assets/javascripts/application.js]: not using it anywhere in the app (yet), so it's just loading pointless data. :|

Comment (bermannoah) [inline app/assets/javascripts/application.js]: makes it more anonymous too

**Pull request merged**